cs.c

```diff
@@ -514,6 +514,23 @@ cs_err CAPSTONE_API cs_close(csh *handle)
 	return CS_ERR_OK;
 }
 
+// replace str1 in target with str2; target starts with str1
+// output is put into result (which is array of char with size CS_MNEMONIC_SIZE)
+// return 0 on success, -1 on failure
+static int str_replace(char *result, char *target, const char *str1, char *str2)
+{
+	// only perform replacement if the output fits into result
+	if (strlen(target) - strlen(str1) + strlen(str2) < CS_MNEMONIC_SIZE - 1)  {
+		// copy str2 to begining of result
```

Comment:
If u know the length of both strings doing strcpy and strcat underperforms

Reply:
You can do memcpy which is faster or memmove if u want to make this function generic and usable from other parts of capstone, because the strreplace funciton name is misleeading because it will only work for this specific case and this specific buffer size